manifests/es-ui-upgrades.yaml

```
@@ -0,0 +1,13 @@
+name: ES-ui-upgrades
+authors: Roger Tabris
+homepage: https://github.com/lumbar527/es-ui-upgrades
+license: GPL-3-or-later
```

```suggestion
license: GPL-3.0-or-later
```
https://spdx.org/licenses/GPL-3.0-or-later.html